gridstatus/spp.py

```diff
@@ -707,6 +707,56 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    @support_date_range("DAY_START")
+    def get_day_ahead_marginal_clearing_prices(self, date, end=None, verbose=False):
```

Comment:
i actually think this dataset name can be better. one sec

Reply:
Keep the function name or update to `get_spp_as_prices`?

Reply:
`get_day_ahead_operating_reserve_prices`